### PR TITLE
fix dummy adapter extra nacls bug

### PR DESCRIPTION
**Pull request by noamhammer**

### CODE REVIEW

File: packages/core/src/core/fetch.ts [hunk 1]
```diff
@@ -336,16 +336,6 @@ const fetchAndProcessMergeErrors = async (
     }
 
     const { errors: mergeErrors, merged: elements } = await mergeElements(awu(serviceElements))
-    // applyInstancesDefaults(elements.filter(isInstanceElement))
```
Comment (roironn) [inline packages/core/src/core/fetch.ts:339]: Meh. Shouldn't be in this PR but OK :)

File: packages/netsuite-adapter/src/adapter.ts [hunk 2]
```diff
@@ -247,6 +247,8 @@ export default class NetsuiteAdapter implements AdapterOperations {
 
   public async deploy({ changeGroup }: DeployOptions): Promise<DeployResult> {
     return this.client.deploy(changeGroup, this.deployReferencedElements)
+    // TODO: fix me
```
Comment (roironn) [inline packages/netsuite-adapter/src/adapter.ts:250]: Shu Hada?

File: packages/salesforce-adapter/src/filters/layouts.ts [hunk 3]
```diff
@@ -74,6 +74,7 @@ const filterCreator: FilterCreator = ({ config }) => ({
     if (layouts.length === 0) {
       return
     }
+    // TODO: fix me
```
Comment (roironn) [inline packages/salesforce-adapter/src/filters/layouts.ts:77]: Shu Hada?

File: packages/workspace/src/workspace/hidden_values.ts [hunk 4]
```diff
@@ -441,7 +441,7 @@ const diffElements = <T extends Element>(visibleElem?: T, fullElem?: T): T | und
 
 // Avoid using this function from out of this file. This filters out only the hidden changes but not
 // their side effects like done in handleHiddenChanges.
-const filterOutHiddenChanges = async (
+export const filterOutHiddenChanges = async (
```
Comment (roironn) [inline packages/workspace/src/workspace/hidden_values.ts:444]: Why is this exported?

**Review by roironn — APPROVED** [review]
Approved, but some things are not a part of this PR (the export and the comments) so they should probably not be in this one... (5 file changes - only 1 is relevant to the actual PR :/)

File: packages/workspace/src/workspace/hidden_values.ts (hunk 4 above)
Comment (noamhammer) [reply packages/workspace/src/workspace/hidden_values.ts:444]: for the SaaS, we need it there (it is like that in the master and we probably removed it in one of our rebases)

File: packages/salesforce-adapter/src/filters/layouts.ts (hunk 3 above)
Comment (noamhammer) [reply packages/salesforce-adapter/src/filters/layouts.ts:77]: It means that there is a commented out code below and we need to figure out why we commented it out and fix that (either remove the code or to uncomment it in some way)

**Pull request merged**